src/components/LoadingSmall.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Loader, Segment } from 'semantic-ui-react'
+import { SemanticSIZES } from 'semantic-ui-react'
+
+const LoadingSmall: React.FC<{size?: SemanticSIZES}> = ({size ='small'}) => {
+  return (
+    <Segment basic style={{ height: '50%', margin: 0}}>
```

Comment:
```suggestion
    <Segment basic>
```
Remember we'd already changed this.

Reply:
Thanks Carl. I managed to lose those changes, and was unable to find them again.